test/demo/basic2.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
```

Comment:
Do we need this? If so, we should rename it to something more meaningful.

Reply:
My bad, I did not want to add this. It was just to test the `maxTilesPerFrame` behavior, which then I increased to `10` for better behavior on load time. I will try to test this once more and remove unecessary changes.